lib/api/webSockets.js

```diff
@@ -42,6 +44,23 @@ const webSocketsFactory = ({config, logger}) => {
 
 			const content = path(['content'], obj) || {};
 
+			if (content.type === 'requestLog') {
```

Comment:
I would also add a condition here to emit events only if there are listeners. To avoid extra processing. Some logs can be quite big.

See https://nodejs.org/api/events.html#events_emitter_listenercount_eventname

Something like:
```
if (self.listenerCount('networkLog')) { // ...
```